app/components/components.md

```diff
@@ -0,0 +1,141 @@
+**ALL PAGES**
+
+* NAV component:
+- site name
+- cart icon
+- login button || username
+
+* FOOTER component?
+
+**PRODUCTS PAGE**
+
+* FILTER component:
+- Categories
+- Price ranges
+- keyword
+
+* ALL PRODUCTS component:
+- pagination
+- sort by
+- image
+- title
+- price
+- '+' (add to cart)
+- more info
+
+**INDIVIDUAL PRODUCT PAGE**
+
+* PRODUCT component:
+- image
+- title
+- price
+- description
+- add to cart (disabled if product is sold out)
+
+**CART PAGE**
+
+* CART component:
+- cart items
+- total cost
+- checkout
+
+* CARTITEM component:
+- image
+- title
+- price
+- quantity (editable)
+
```

Comment:
Think we should have some kind of "Checkout Success" page once a user hits the checkout button.
It would just show the new order that's been created and then have a link back to the home page.

Reply:
agreed, done